fish-be/src/main/game/player.ts

```diff
@@ -38,17 +37,39 @@ export class Player implements IPlayer {
     return false;
   }
 
-  removeCard(card: Card): boolean {
+  stealCardFrom(player: Player, card: Card): boolean {
+    // ensure player can steal from the halfsuit
+    let halfSuit = false;
+    for (const c of this._hand) {
+      if (c.isSameHalfSuit(card)) {
+        halfSuit = true;
+      }
+    }
+    console.log('halfsuit');
```

Comment:
Why are we logging 'halfsuit' here? was it for debugging purposes?

Reply:
Yes, it's removed